WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -34,7 +34,7 @@ struct TotalsView: View {
                     Spacer()
                         .renderedIf(cardReaderViewLayout.topPadding == nil)
 
-                    VStack(alignment: .center, spacing: Constants.verticalSpacing) {
+                    VStack(alignment: .center, spacing: dynamicTypeSize.isAccessibilitySize ? 0 : Constants.verticalSpacing) {
```

Comment:
For larger non-accessibility sizes, the button may not fit.

This is the only blocking comment. The other ones are optional improvements.

![IMG_0156](https://github.com/user-attachments/assets/c823fca6-a295-4c87-8bc6-85e69d3b87d6)

Reply:
That's a very good catch, I've updated the dynamic spacing so it's more nuanced and takes into account non-accessibility sizes as well: https://github.com/woocommerce/woocommerce-ios/pull/14935/commits/16cff5f4ec75faa038d36921e338c02943991f81 and it shows correctly for x+ sizes before moving to accessibility sizes:

| xlarge | xxlarge | xxxlarge |
|--------|--------|--------|
| ![_xlarge](https://github.com/user-attachments/assets/9fe34456-9be9-477a-8d28-4e14864ff15f) | ![_xxlarge](https://github.com/user-attachments/assets/373b9748-8fdc-4123-a96c-67282d8e0cd0) | ![_xxxlarge](https://github.com/user-attachments/assets/1cfffed9-99ad-46d8-8edb-78dc94271f23) | 

I'll check the other items now and test on iPad Mini as well to fine-tune this a bit better where possible 👍